### PR TITLE
fix changelog entries, add v1.28.0

### PR DESCRIPTION
**Pull request by laurenolivia**
## Description

Fix changelog entries. Add `v1.28.0`.

## Output from tests
Including output from tests may require access to a TFE instance. Ignore this section if you have no environment to test against.

```
$ TFE_ADDRESS="https://example" TFE_TOKEN="example" go test ./... -v -run TestFunctionsAffectedByChange

...
```

### CODE REVIEW

**Review by brandonc — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,24 @@
 # UNRELEASED
 
-# v1.27.0
+# v1.28.0
 
 ## Enhancements
-* Adds `RunPreApplyRunning` and `RunQueuingApply` run statuses by @uk1288 [#712](https://github.com/hashicorp/go-tfe/pull/712)
 * Update `Workspaces` to include associated `project` resource by @glennsarti [#714](https://github.com/hashicorp/go-tfe/pull/714)
 * Adds BETA method `Upload` method to `StateVersions` and support for pending state versions by @brandonc [#717](https://github.com/hashicorp/go-tfe/pull/717)
+* Adds support for the query parameter `q` to search `Organization Tags` by name [#720](https://github.com/hashicorp/go-tfe/pull/720)
```
Comment (sebasslash) [inline CHANGELOG.md:8]: Update `name` to the correct username here 😄

**Review by sebasslash — CHANGES_REQUESTED** [review]
One minor thing before merge:

File: CHANGELOG.md (hunk 1 above)
Comment (laurenolivia) [reply CHANGELOG.md:8]: What do you mean by username?
The Organization Tag attribute is `name` according to the docs [here](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/organization-tags#list-tags)

Comment (laurenolivia) [reply CHANGELOG.md]: Oh nvm I see what you mean

**Review by brandonc — APPROVED** [review]

**Pull request merged**